index.html

```diff
@@ -39,8 +39,209 @@ <h1 class="hero-title" >I’m Mariane <br>
+                    </article>
+                </aside>
+            </article>
+                <!-- work 4  -->
+            <article class="work ">
+                <img class="work-main-img" src="./assets/Snapshoot Portfolio4.svg" alt="">
+                <aside class="work-info">
+                    <h2 class="work-title" >Multi-Post Stories</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
+                <p class="hero-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <a href="#">LET'S CONNECT</a>
+
+                <aside class="social-media">
+                    
+                    <a href="">
+                        <img src="./assets/social media icons.svg" alt="">
+                    </a>
+                </aside>
+
+                <article class="work-button work-button-about">
+                    <button class="btn" > Get my resume </button>
+                </article>
+            </article>
+            
+        </div>
+        <article class="languajes">
+            
+            <div class="languajes-title-cont">
+                <h2>Languages</h2>
+                <img src="./assets/arrow-down.svg" alt="">
+            </div>
+
+            <ul class="languajes-cont">
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 1.svg" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="languaje-item" >
```

Comment:
- [ ] [**OPTIONAL**] The previous CR requested some change, I noticed that they were not too much clear. But I think you should change the border bottom of this `li`'s so they stick to the figma design :+1: . Kindly update that before submitting, because it's an important par of the requirements.

![Screenshot from 2023-02-17 19-57-06](https://user-images.githubusercontent.com/80458199/219740761-1ada81c6-df9b-47d9-a4b9-113ddeac62dc.png)

![219716950-90e337ce-f26d-46e3-bde0-ad07e97bbec6](https://user-images.githubusercontent.com/80458199/219741058-5dd0345a-f7bc-4c9e-a3d5-688884c6a420.jpg)